source/streaming-mode/streaming-read-config.txt

```diff
@@ -0,0 +1,188 @@
+.. _spark-streaming-read-conf:
+
+====================================
+Streaming Read Configuration Options
+====================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. _spark-streaming-input-conf:
+
+You can configure the following properties when reading data from MongoDB in streaming mode.
+
+.. include:: /includes/conf-read-prefix.rst
+
+.. list-table::
+   :header-rows: 1
+   :widths: 35 65
```

Comment:
I don't think we have control over this, but it's interesting that this table doesn't render "striped" like it does on the batch read page. Definitely makes it a little harder to read